pom.xml

```diff
@@ -171,6 +161,12 @@
             <version>2.3.0</version>
         </dependency>
 
+		<dependency>
+		    <groupId>com.google.code.gson</groupId>
+		    <artifactId>gson</artifactId>
+		    <version>2.10</version>
+		</dependency>
+
```

Comment:
You should check the diff of the pom.xml file on this branch with `gtmaster`.  I think the only changes should be this new `gson` dependency, and the version number (`3.2.0-SNAPSHOT` instead of `3.1.2-SNAPSHOT`).